lib/puppet/type/file/ensure.rb

```diff
@@ -175,6 +183,14 @@ def retrieve
     end
 
     def sync
+      if self.should == :absent
+        # Make a dummy content property so that we can
```

Comment:
I think I understand this, but the comment could be clearer. What's the content property that exists here otherwise, and how is it set?